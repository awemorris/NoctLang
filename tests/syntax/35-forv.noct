func main() {
    bullets = [];
    bullets->push({ x: 100, y: 100, vy: -400 });
    fast_gc();
    foo();
}

func foo() {
    bullets->push({ x: 100, y: 200, vy: -400 });

    print("bullets.length = " + bullets.length);
    var aliveBullets = [];
    for (b in bullets) {
        print(b);
    }
}
