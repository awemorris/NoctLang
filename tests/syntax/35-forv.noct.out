bullets.length = 2
{vy: -400, y: 100, x: 100}
{vy: -400, y: 200, x: 100}
